Dockerfile

```diff
@@ -1,13 +1,27 @@
-FROM golang:1.11.6-stretch as golang
+ARG BUILDPLATFORM=linux/amd64
+
+FROM --platform=$BUILDPLATFORM golang:1.11.6-stretch as golang
 WORKDIR /go/src/github.com/buoyantio/bb
 ADD .  /go/src/github.com/buoyantio/bb
 
 RUN mkdir -p /out
 RUN ./bin/dep ensure
-RUN go build -o /out/bb .
+ARG TARGETARCH
+RUN GOOS=linux GOARCH=$TARGETARCH go build -o /out/bb .
+
+FROM debian:buster-20200514-slim
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
+    dnsutils \
+    iptables \
+    jq \
+    nghttp2 \
+    ca-certificates && \
```

Comment:
What do you think of moving the `&&` to the same line as `rm -rf`, like we had [here](https://github.com/linkerd/linkerd2/blob/stable-2.7.1/Dockerfile-base#L15)? For me it's more readable, but take it or leave it